dsl/jpql/src/test/kotlin/com/linecorp/kotlinjdsl/dsl/jpql/LocalDateTimeExpressionDslTest.kt

```diff
@@ -0,0 +1,25 @@
+package com.linecorp.kotlinjdsl.dsl.jpql
+
+import com.linecorp.kotlinjdsl.querymodel.jpql.expression.Expression
+import com.linecorp.kotlinjdsl.querymodel.jpql.expression.Expressions
+import org.assertj.core.api.Assertions.assertThat
+import org.junit.jupiter.api.Test
+import java.time.LocalDateTime
+
+class LocalDateTimeExpressionDslTest {
+
+    @Test
+    fun `localDateTime to support LOCAL DATETIME in jpql`() {
```

Comment:
To keep test names simple, we only write the name of the target method in the test name unless there are multiple test cases. 

![image](https://github.com/line/kotlin-jdsl/assets/31586979/372af0e1-5b63-4bb8-bbe6-521548e65171)